charts/identity-server/values.dev.yaml

```diff
@@ -6,7 +6,7 @@ auth-admin-web:
     NEXTAUTH_URL: 'https://identity-server.dev01.devland.is/admin/api/auth'
     NEXT_PUBLIC_BACKEND_URL: '/backend'
     NODE_OPTIONS: '--max-old-space-size=208'
-    SERVERSIDE_FEATURES_ON: ''
+    SERVERSIDE_FEATURES_ON: 'delegation-api'
```

Comment:
Out of curiosity what how do you add more features to SERVERSIDE_FEATURES_ON?
Is it a comma separated list or an array or just not supported?

Reply:
This is generated automatically from here: https://github.com/island-is/island.is/pull/8753/files/c017d3ccf78585496a0b7351f495a6a23fff102d#diff-f7be9b6be4760aab04ee07d7a80b2ded5b8338cbdb39c4e77ed852749b7c8a4e

Note that there it is an array so I assume it gets converted to comma seperated string or something like that.